/*
 This program is free software; you can redistribute it and/or modify it under
 the terms of the European Union Public Licence - EUPL v.1.1 as published by
 the European Commission.

 This program is distributed in the hope that it will be useful, but WITHOUT
 ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 FOR A PARTICULAR PURPOSE. See the European Union Public Licence - EUPL v.1.1
 for more details.

 You should have received a copy of the European Union Public Licence - EUPL v.1.1
 along with this program.

 Further information about the European Union Public Licence - EUPL v.1.1 can
 also be found on the world wide web at http://ec.europa.eu/idabc/eupl

*/

/*
 ------ Copyright (C) 2011 STA Steering Board (space.trajectory.analysis AT gmail.com) ----
*/

/*
------------------ Author: Guillermo Ortega  -------------------------------------------
*/


#include "ui_stoppingConditionsTrajectory.h"

#include <QDialog>

class stoppingConditionsTrajectoryDialog : public QDialog , private Ui::stoppingConditionsTrajectoryDialog
{
    Q_OBJECT
public:
	stoppingConditionsTrajectoryDialog( QWidget * parent = 0, Qt::WindowFlags f = 0 );
	~stoppingConditionsTrajectoryDialog();

protected slots:
//	void on_stoppingConditionsTrajectoryDialog_accepted();
//	void on_stoppingConditionsTrajectoryDialog_rejected();
//	void on_stopCondValueEdit_textChanged(const QString&);
//	void on_stopCondTolEdit_textChanged(const QString&);
//	void on_stopCondComboBox_currentIndexChanged(int);
//	void on_stopCondAddButton_clicked(bool);
//	void on_stopCondCentralBodyCombobox_currentIndexChanged(int);
//	void on_stopCondRemoveButton_clicked(bool);
	
};

